index.html

```diff
@@ -1325,6 +1327,44 @@ <h2>
 detail how revocation is performed and tracked.
       </p>
 
+      <p>
+All public key formats MUST be expressed in either JSON Web Key (JWK)
+format using the <code>publicKeyJwk</code> property or one of the native
```

Comment:
Either replace the word "native" with "other" or "additional" or simply delete it.  (Native isn't correct because the native key formats used by cryptographic algorithms are all binary.)

Reply:
Done.